python/docs/intro.rst

```diff
@@ -0,0 +1,93 @@
+to work with dlis-files without having to know the details of the standard
+itself. Its main focus is making the data accessible while putting little
+assumptions on how the data is to be used. 
+
+dlisio is written and maintained by Equinor ASA as a free, simple, easy-to-use
+way of reading well logs that can be tailored to our needs, and as a
+contribution to the free software community.
+
+Please note that dlisio is still in alpha, so expect breaking changes between
+versions.
+
+Digital Log Interchange Standard (DLIS)
+=======================================
+
+dlis is a binary file format for well logs, developed by Schlumberger in the
+late 80's and published by the American Petroleum Institute (API) in 1991. It
+is now the Petroleum Open Standards Consortium (POSC) [2]_ that has the
+stewardship. 
+
+When developing dlis standard, the main emphasis was put on easy writing. The
+files are structured in such a way that files can be written directly while
+acquiring the logs. This is very handy for the producers of the files, and
+equally tedious for the consumer that want to read them later on.
+Additionally is a very tolerant standard. It specifies general data-structures
+within the file, such as channels and frames, but allow the producers to modify
+these to fit their needs. It also allow for completely new structures, not
+defined by the standard itself, such as vendor-specific object-types. This all
+sums up to a fairly complex standard with a lot of quirks and weirdness to it.
+It is safe to say that dlis a particular difficult format to work with.
+
+Physical and logical files
+--------------------------
+
+The dlis standard distinguish between physical files and logical files. A
+physical file would be a .dlis file. A logical file on the other hand, is a
+collection of data that are logically connected. Typically this can be all
+measurements and metadata gathered at one run. A physical file may contain
+multiple logical files, where each logical file is completely independent of
+the other logical files:: 
+    
+                            physical file (.dlis)
+            --------------------------------------------------------
+           | logical file 0 | logical file 1 | ... | logical file n |
+            --------------------------------------------------------
+
+dlisio's main entrypoint, :py:func:`dlisio.load()` takes a physical file and
+returns a tuple-like object with all the logical files.
+
+Logical files can be thought of as a pool of metadata and curve-values that all
+are related in some way. Some common metadata objects are Fileheader, Origins,
+Tool, Parameter and Calibration. For a full list of all metadata objects. See
+all object-type :ref:`Object types`::
+
+                            logical file         
+            ------------------------------------------------------
+           | Fileheader | Origin | Frame1 | Channel | Frame1 | .. |
```

Comment:
This looks weird to me because of the Frame - do you mean Frame EFLR or Fdata? And why both are Frame1?
I am not sure you can make it user-friendly
I would suggest something like Fileheader - Origin - [meta]Channel - [meta]Frame - FrameData1 - FrameData2

Reply:
The difference between FRAME EFLR and FDATA is abstracted away by dlisio and frankly the end user does not need to know about this. Its stated below that this is a simplified illustration of a logical record. It is there to illustrate to the user how think about logical files when working with dlisio, not how LF's are implemented in the standard